SU-HSDO/suhumsci
# Feat(STN-597): update figcaption to new caption/credit styling in WYSIWYG

# READY FOR REVIEW

## Summary
This updates the old `figcaption` styling to match our new universal caption/credit styling as a gradient overlay.

"There are 2 caption fields for text area images. One is in the media library, the other is added via the WYSIWYG. This ticket’s scope is limited to displaying the caption entered in the WYSIWYG (not the one stored in the media library). "

[JIRA Card: 597](https://sparkbox.atlassian.net/browse/STN-597)

## Need Review By (Date)
2/3/21

## Urgency
Medium

## Steps to Test
1. `npm run test`
2. To test this new styling you'll need to add a page using the image property caption spot, this work does not style the caption/credit that is coming from the media module caption entry area. This option will show up when you add an image through the WYSIWYG or you can click the "Edit Media" button on an existing image you've already placed.
![image](https://user-images.githubusercontent.com/9355698/106677835-7736c300-6587-11eb-99fa-e2566f647983.png)
3. The figcaption should now resemble the same look and feel of the caption work we've done with the postcards and earlier elements. Where the text is white and over a dark gradient overlay at the bottom of the images.
4. Changing the alignment of the image should not change the caption's function, it should stay with the image.
5. Changing the image style such as "Small Scaled" shouldn't affect the caption application either, it should scale and move with the image size.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
